io.github.kraflab.DSDA-Doom.yml

```
@@ -0,0 +1,35 @@
+id: io.github.kraflab.DSDA-Doom
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: dsda-doom
+finish-args:
+- --device=all
+- --share=ipc
+- --socket=wayland
+- --socket=fallback-x11
+- --socket=pulseaudio
+modules:
+- "shared-modules/glu/glu-9.json"
+- "shared-modules/linux-audio/fluidsynth2.json"
+- name: libzip
+  buildsystem: cmake-ninja
+  config-opts:
+  - -DCMAKE_BUILD_TYPE=Release
+  sources:
+  - type: git
+    url: https://github.com/nih-at/libzip
+    tag: v1.11.3
+- name: dsda-doom
+  buildsystem: cmake-ninja
+  subdir: prboom2
+  config-opts:
+  - -DCMAKE_BUILD_TYPE=Release
+  sources:
+  - type: git
+    url: https://github.com/kraflab/dsda-doom
+    tag: v0.28.3
+  post-install:
+  - "desktop-file-edit --set-icon=${FLATPAK_ID} /app/share/applications/dsda-doom.desktop"
+  - "mv /app/share/icons/hicolor/scalable/apps/{dsda-doom,$FLATPAK_ID}.svg"
+  - "mv /app/share/applications/{dsda-doom,$FLATPAK_ID}.desktop"
```

Also `rename-desktop-file: dsda-doom.desktop` and

```suggestion
  - "mv /app/share/applications/{dsda-doom,$FLATPAK_ID}.desktop"
```